__tests__/hooks/useGroupSetting.test.js

```diff
@@ -1,9 +1,8 @@
-import * as launchGallery from 'react-native-image-picker';
 import React from 'react';
-import SimpleToast from 'react-native-simple-toast';
+import * as launchGallery from 'react-native-image-picker';
 import {act, renderHook} from '@testing-library/react-hooks';
 
-import * as serviceFile from '../../src/service/file';
+import {Linking} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed and `SimpleToast` from 'react-native-simple-toast' has been removed. If this package is no longer used in the file, it's a good practice to remove unused imports. However, if it's still needed somewhere in the code not shown in the hunk, this could cause an error.
```diff
- import SimpleToast from 'react-native-simple-toast';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->